File: app/src/main/java/com/qns/data/local/entity/ChatEntity.java
package com.qns.data.local.entity;
import androidx.annotation.NonNull;
import androidx.room.*;
@Entity(tableName = "chats")
public class ChatEntity {
    @PrimaryKey @NonNull public String id = "";
    public String type, name, otherUserId, otherUsername, otherUserScamReason;
    public long lastMessageAt, createdAt;
    public boolean otherUserOnline = false, otherUserScam = false, otherUserVerified = false;
    public int     unreadCount = 0;
}

File: qunes-android-app/src/main/kotlin/com/qunes/app/data/local/entities/ChatEntity.kt
package com.qunes.app.data.local.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "chats")
data class ChatEntity(
    @PrimaryKey val id: String,
    val name: String?,
    val isGroup: Boolean,
    val lastMessage: String?,
    val updatedAt: Long
)